Runtime/Triangulator.cs

```diff
@@ -1804,7 +1800,8 @@ public void Execute(Allocator allocator, bool refineMesh, bool constrainBoundary
                     return;
                 }
 
-                if (!utils.SupportsMeshRefinement) {
+                if (!utils.SupportRefinement())
```

Comment:
`SupportRefinement` is not correct English for this use case. That would be `SupportsRefinement` (see original wording).

Reply:
I believe both of the wording are correct. Rather minor issue.